tomcat/src/main/java/org/apache/coyote/http11/Http11Processor.java

```diff
@@ -28,19 +37,36 @@ public void process(final Socket connection) {
         try (final var inputStream = connection.getInputStream();
              final var outputStream = connection.getOutputStream()) {
 
```

Comment:
```suggestion
        try (final var inputStream = connection.getInputStream();
             final var outputStream = connection.getOutputStream();
             final var bufferedReader = new BufferedReader(new InputStreamReader(inputStream))) {

```

BufferedReader도 close 되도록 try with resources 구문에 관리되게 하는 것이 좋을 것 같아요

Reply:
(이펙티브 자바 스터디에서 봤는데) 수정하겠습니다